docs/oscar_references.bib

```diff
@@ -812,6 +812,13 @@ @Article{Kem99
   url           = {https://doi.org/10.1006/jsco.1998.0247}
 }
 
+@Misc{Kin07,
+  author        = {King, Simon},
+  title         = {Fast computation of secondary invariants},
+  howpublished  = {arXiv:math/0701270},
+  year          = 2007
```

Comment:
Please add the DOI, it is `10.48550/arXiv.math/0701270`

(We probably should review all our arXiv refs and make them uniform in this regard,